### ISSUE TITLE
c2d of improper transfer function

### ISSUE THREAD

**Issue by mfalt** [JuliaControl/ControlSystems.jl#128]
c2d of improper transfer function breaks silently since it goes through state-space, and isproper is broken, see https://github.com/JuliaControl/ControlSystems.jl/issues/85
Should we enable some way of discretizing a transferfunction other than going through state-space form?

**Comment by olof3** [comment]
Yes, at least it should be possible to do pole-zero matching for systems on zpk form.
